test-cypress/steps/repository-steps.js

```diff
@@ -234,4 +254,16 @@ export class RepositorySteps {
     static getLocalGraphDBTable() {
         return cy.get('#wb-locations-locationInGetLocations');
     }
+
+    static clickObdaSelectFilesBtn() {
+       cy.get('.selectFiles').first().click();
+    }
+
+    static uploadObdaFile(file) {
+        cy.get('input[type=file]').first().selectFile(file, {force: true});
```

Comment:
Why `force:true`? You should always try to avoid using `force`, because it migth cover up some bug in the UI. If it's not possible to avoid it's usage, then you must document why it was needed.

Reply:
Will add a comment for the need of `force` here.